src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -52,7 +105,10 @@
    (comidi/GET "/environments" request
                (request-handler request))
    (comidi/GET ["/status/" [#".*" :rest]] request
-               (request-handler request))))
+               (request-handler request))
+
+   (comidi/GET ["/environment_classes" [#".*" :rest]] request
+               ((environment-class-handler jruby-service) request))))
```

Comment:
Good call, thanks.  That would be a better way to do it.  I'll make that change.